packages/block-editor/src/components/inspector-popover-header/index.js

```diff
@@ -31,8 +31,7 @@ export default function InspectorPopoverHeader( {
 				<Spacer />
 				{ actions.map( ( { label, icon, onClick } ) => (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
**Before:**
![inspector-popover-header-before](https://github.com/user-attachments/assets/7debbc9d-3381-4820-821d-8d96d1bfdff9)

**After:**
![inspector-popover-header-after png](https://github.com/user-attachments/assets/d2ce9026-3573-4924-945f-95bc88b9660a)

Reply:
Could we consider removing the style override, and setting `size="small"` instead?

Reply:
Yup! Part of my suggestion above was also to [remove the style overrides](https://github.com/WordPress/gutenberg/blob/05e05ae997093197350b9f845696bb757c0e4384/packages/block-editor/src/components/inspector-popover-header/style.scss#L5-L16)

Reply:
Just to make sure, I am suggesting that we delete [these styles](https://github.com/WordPress/gutenberg/blob/05e05ae997093197350b9f845696bb757c0e4384/packages/block-editor/src/components/inspector-popover-header/style.scss#L5-L16)